src/components/SlateEditor/RichTextEditor.jsx

```diff
@@ -14,13 +14,27 @@ import { Editor } from 'slate-react';
 import { isKeyHotkey } from 'is-hotkey';
 import BEMHelper from 'react-bem-helper';
 import { css } from '@emotion/core';
+import { hasNodeOfType } from './utils';
 import createSlateStore, { setSubmitted } from './createSlateStore';
 import { PluginShape } from '../../shapes';
 
 const isBoldHotkey = isKeyHotkey('mod+b');
-const isItalicHotkey = isKeyHotkey('mod+i');
-const isUnderlinedHotkey = isKeyHotkey('mod+u');
```

Comment:
Underline burde vi vel fortsatt ha med?

Reply:
Nei det er fjerna fra menyen for lenge siden. Argumentet var å skille mellom lenker og underline. https://trello.com/c/3KP7sq45/366-fjerne-mulighet-for-understreking-i-br%C3%B8dtekst